src/Identifier/LdapIdentifier.php

```diff
@@ -30,9 +30,14 @@
  * ```
  *  new LdapIdentifier([
  *       'host' => 'ldap.example.com',
- *       'bindDN' => function($username) {
- *           return $username; //transform into a rdn or dn
- *       },
```

Comment:
Changing `bindDN` from a callable into a string looks like a breaking change. Is there a way we accommodate both the current configuration options and add in the new filtering as an optional layer?